database/cockroachdb/README.md

```diff
@@ -1,19 +0,0 @@
-# cockroachdb
-
-`cockroachdb://user:password@host:port/dbname?query` (`cockroach://`, and `crdb-postgres://` work, too)
-
-| URL Query  | WithInstance Config | Description |
-|------------|---------------------|-------------|
```

Comment:
Can you keep this ?Might be useful as we have an investigation for cockroach planned at some point

Reply:
We can always refer back to the original source if we need it. I'd prefer to not have to maintain the tests and everything that we don't support. 

I'm just finding a lot problems like poor internal contracts on the Handler type and the testing package failing on different sources and backends, and leaving running docker containers around.